spec/features/maintaining_a_visit_spec.rb

```diff
@@ -1,15 +1,18 @@
 require 'rails_helper'
 
-VCR.configure do |config|
-  config.default_cassette_options = {
-    match_requests_on: %i[ method uri host path body ]
-  }
-end
-
 RSpec.feature 'Maintaining a visit', js: true do
   include FeaturesHelper
 
-  scenario 'viewing and withdrawing a visit request', vcr: { cassette_name: :maintianing_a_visit_viewing_and_withdrawing } do
+  before do
+    RequestStore.store[:request_id] = 'myRequestID123'
+  end
+
+  custom_matchers = %i[ method uri host path valid_uuid ]
+
+  scenario 'viewing and withdrawing a visit request', vcr: {
```

Comment:
Could we add the `valid_uuid` to the VCR default config (`config.default_cassette_options`) so we can avoid having to configure all the scenarios individually?